manifest.yml

```diff
@@ -72,7 +72,20 @@ dependencies:
     cf_stacks:
       - cflinuxfs2
     uri: https://pivotal-buildpacks.s3.amazonaws.com/concourse-binaries/python/python-3.5.2-linux-x64.tgz
-    md5: 701a135b3228075c2ce59c527bc92e1e
+    md5: 701a135b3228075c2ce59c527bc92e1e    
+- name: python
```

Comment:
I think you need to indent lines 76-87 one level to the right (probably two spaces) so that they have the same indentation as the other dependencies.